### DISCUSSION TITLE
schema question

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#149] [Q&A]
Hi there, now we have rgb20, and rgb21 script code here:

<img width="413" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/6e563661-5766-4fa8-a1b6-98fe46899f8e">
<img width="525" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/0aff0bf2-d4f0-400b-99ba-03fb3f296604">
So, my question is: For each specific schema, we need a script to run it, right? (Like rgb20, we need a nia_schema code, and rgb21, we need a uda_schema code in the rgb-schemata). 
So if I want to add a new rgb22 schema(or customize schema), then I need to write a script code to run it.

**Comment by dr-orlovsky** [comment]
You need the script only if you have a validation logic which can't be expressed in Schema itself. Otherwise you can have it empty
